debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/PostgresConnectorIT.java

```diff
@@ -2509,6 +2509,69 @@ public void shouldThrowWhenTableFiltersIsEmpty() throws Exception {
         });
     }
 
+    private List<Long> getSequence(SourceRecord record) {
+        assertTrue(record.value() instanceof Struct);
+        Struct source = ((Struct) record.value()).getStruct("source");
+        String stringSequence = source.getString("sequence");
+        List<Long> sequence = new ArrayList<>();
+        assertEquals('[', stringSequence.charAt(0));
+        assertEquals(']', stringSequence.charAt(stringSequence.length() - 1));
+        for (String s : stringSequence.substring(1, stringSequence.length() - 1).split(",")) {
+            sequence.add(Long.parseLong(s.trim()));
+        }
+        return sequence;
+    }
```

Comment:
This comes back to the discussion around JSON. I suppose this could be a one-liner give or take when consuming it with Jackson or similar?